gradle.properties

```diff
@@ -1,3 +1,8 @@
 org.gradle.parallel=true
 org.gradle.jvmargs=-Xmx2g
 org.gradle.caching=true
+org.gradle.vfs.watch=true
+
+# Gradle will use all CPU cores by default. A Dev can tune the workers.max property to control the number of cores Gradle uses to suit their needs.
+# i.e. Give Gradle all cores if wanting to build as fast as possible. Setting this to (# of cores - 1) is a good default.
+org.gradle.workers.max=3
```

Comment:
This is 3 for me as I'm on an older laptop :P 

It seems like a sane default since laptop these days usually ship with 4 cores.